vertical-pod-autoscaler/hack/deploy-for-e2e.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+
+REGISTRY=gcr.io/`gcloud config get-value core/project`
+TAG=latest
+
+COMPONENTS="recommender updater admission-controller"
+
+if [ $# -ne 0 ]; then
+  COMPONENTS=$*
+fi
+
+for i in ${COMPONENTS}; do
+  (
+    cd ${SCRIPT_ROOT}/${i} && make release
```

Comment:
Hint:
```
make --directory ${i} release
```
It's a nice shortcut that also goes back to your previous directory afterwards.